model/project_ref.go

```diff
@@ -1420,7 +1429,11 @@ func IsPerfEnabledForProject(projectId string) bool {
 	if err != nil || projectRef == nil {
 		return false
 	}
-	return utility.FromBoolPtr(projectRef.GetProjectParserMergedProjectRef("").PerfEnabled)
+	mergedProjectRef, err := projectRef.MergeWithParserProject("")
+	if err != nil || mergedProjectRef == nil {
+		return false
+	}
+	return utility.FromBoolPtr(mergedProjectRef.PerfEnabled)
```

Comment:
FYI We have an IsPerfEnabled (and similar functions) to make usage more readable but either work